gcp/rakefiles/secrets.rb

```diff
@@ -96,13 +101,19 @@ def self.set_secrets(collected_secrets, exekube_cmd)
           populated_secrets = {}
           secrets.each do |secret_name|
             if ENV[secret_name.upcase].to_s.empty?
-              secret_value = SecureRandom.hex
+              if secret_name.match(/^(key)_/)
```

Comment:
How about using `startswith?` instead of all these regular expressions?

Reply:
I like regular expressions (spent a lot of time writing smart html parsers), I am fine with them. But if we decide not to use them, I am also fine.

Reply:
Regular expressions are great, but I avoid them unless they are necessary since they are harder to read and more computationally expensive than a String method like `startswith?`.

Reply:
I have no opinion on that, I don't think computational expenses matter here, but whatever makes more sense to you.